packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/list/component.js

```diff
@@ -5,8 +5,12 @@ export default {
   bindings: {
     listeners: '<',
     goToListenerCreation: '<',
+    getListenerEditionLink: '<',
     goToListenerEdition: '<',
     goToListenerDeletion: '<',
+    getPoolDetailLink: '<',
+    listenerEditionTrackingHit: '<',
```

Comment:
**nitpick(seperation-of-concerns):** take `trackBase` as input to the component and define `edit` related base in the edit component itself